doc/_static/custom_styles.css

```
@@ -39,6 +39,14 @@ div.fasb h3 {
   padding-bottom: 0.3em;
 }
 
+/* Center iframes and adjust their bottom margins */
+iframe {
+  display: block;
+  margin-left: auto;
+  margin-right: auto;
+  margin-bottom: 1.5rem;
+}
```

```suggestion
  margin-top: -0.7rem;
}
```

Add negative top margin to reduce height above videos

---

Agreed

---

I was thinking to do this but I decided not to because the current `margin-top` for iframes is the same as the one for images.

So, should we use this  `margin-top` for images too?

---

I guess yes, but I think that should be in another PR. For now we can fix the one in the video frames. What do you think @CAM-Gerlach ?

---

I think I found the issue: it turns out we're using our main website's css here, which is causing a lot of other problems.

So, for now I'd suggest to merge this one as it is. Then @dalthviz or me can fix that other problem in a different PR.

---

Agreed, that makes sense. That would explain a lot of the issues we've been seeing with margins on a lot of elements.

---

After the latest changes in the theme, we really need to increase `margin-top` instead of decrease it. @CAM-Gerlach and I think that  `1.0rem` looks good.

@juanis2112, please see how things look on Netlify and let us know what you think.